bankbot.lic

```diff
@@ -427,6 +438,9 @@ arg_definitions = [
 
 args = parse_args(arg_definitions)
 
+# Warn the script user that this is designed to offload items off your person
```

Comment:
This comment seems a bit misleading

Reply:
Copy/paste error

Reply:
https://github.com/rpherbig/dr-scripts/pull/5551